mock/src/main/java/com/tngtech/keycloakmock/impl/UrlConfiguration.java

```diff
@@ -30,6 +31,14 @@ public UrlConfiguration(@Nonnull final ServerConfig serverConfig) {
     } else {
       this.hostname = serverConfig.getDefaultHostname() + ":" + serverConfig.getPort();
     }
+    if (Objects.requireNonNull(serverConfig.getContextPath()).isEmpty()) {
+      this.contextPath = serverConfig.getContextPath();
```

Comment:
Just to make sure, "/" should be handled the same way as "":
```suggestion
    if (Objects.requireNonNull(serverConfig.getContextPath()).isEmpty() || "/".equals(serverConfig.getContextPath())) {
      this.contextPath = "";
```